tests/unit/utils/test_utils.py

```diff
@@ -644,6 +644,16 @@ def test_sanitize_filename_empty_replacement():
     assert utils.sanitize_filename(name, replacement=None) == 'Bad File'
 
 
+@hypothesis.given(filename=hypothesis.strategies.text(min_size=100),
+                  suffix=hypothesis.strategies.text())
+def test_sanitize_filename_invariants(filename, suffix):
+    try:
+        sanitized = utils.sanitize_filename(filename, optional_suffix=suffix)
+    except ValueError:
+        hypothesis.assume(False)
```

Comment:
`hypothesis.assume(False)` [tells](https://hypothesis.readthedocs.io/en/latest/details.html#hypothesis.assume) hypothesis we don't want examples like this, which I don't think is true - we do want examples like that, because they did end up finding a bug in our function :wink: So this should probably just be `pass`?